lamindb/models/schema.py

```diff
@@ -426,13 +471,16 @@ def from_values(  # type: ignore
         Raises:
             ValidationError: If some values are not valid.
 
-        Examples:
+        Example::
```

Comment:
Just a note: I recently realized that only `Examples:` is recognized as a subheading. It looks much prettier than `Examples::`.

But I don't know how to achieve the code formatting then. 🤔